portafly/portafly_api.yml

```diff
@@ -14,28 +14,103 @@ paths:
-        type: integer
-      in: query
-      required: false
-    - name: per_page
-      description: Number of results per page. Default and max is 500.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: state
-      description: Account state.
-      schema:
-        type: string
-      in: query
+      - name: page
+        description: Page in the paginated list. Defaults to 1.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: state
+        description: Account state.
+        schema:
+          type: string
+        in: query
+  /users:
+    summary: A list of user
+    description: A list of users
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/userList'
+      operationId: userList
+      summary: Get account users list
+    parameters:
+      - name: page
+        description: Current page for pagination
+        schema:
+          type: integer
+        in: query
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+  /accounts/invitations:
+    summary: A list of sent invitations
+    description: List of invitations sent by an account
+    get:
+      responses:
+        '200':
+          $ref: '#/components/responses/invitationList'
+      operationId: invitationList
+      summary: Get sent invitations list
+    parameters:
+      - name: page
+        description: Page in the paginated list. Defaults to 1.
+        schema:
+          type: integer
+        in: query
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+      - name: account_id
+        description: Id of current account.
+        schema:
+          type: integer
+        in: query
+  /accounts/invitations/new:
+    summary: A new invitation request
+    description: A new invitation request
+    post:
+      responses:
+        '201':
+          $ref: '#/components/responses/invitationNew'
+      operationId: invitationNew
+      summary: Post a new invitation
+    parameters:
+      - name: email
+        description: Email the invitation will be sent to
+        schema:
+          type: string
+        in: query
+  /accounts/invitations/delete/:id:
+    summary: Deletes an invitation
+    description: Deletes an invitation based on id
+    delete:
+      responses:
+        '204':
+          description: Invitation deleted
+      operationId: invitationDelete
+      summary: Deletes an invitation
+    parameters:
+      - name: id
+        description: ID of invitation to delete.
+        schema:
+          type: integer
+        in: query
```

Comment:
I think the whole invitations is not happening in the end. Maybe @thomasmaas can confirm

Reply:
Removed